third_party/CMakeLists.txt

```diff
@@ -17,30 +17,11 @@ endif()
 # ffmpeg
 ################################################################################
 if (BUILD_FFMPEG)
-  if(WIN32)
-    message(FATAL_ERROR "FFmpeg integration is not supported on Windows.")
-    # TODO
-    # Since pkg-config is not well-supported on Windows (for example, ffmpeg
-    # installed from conda-forge does not include `.pc` medata file),
-    # to support windows, we need to fill interface manually.
-    #
-    # Something like this might work
-    # https://github.com/microsoft/vcpkg/issues/1379#issuecomment-312483740
-  endif()
-  find_package(PkgConfig REQUIRED)
-  pkg_check_modules(LIBAV REQUIRED IMPORTED_TARGET
-    # requires ffmpeg>=4.1
-    libavdevice>=58
-    libavfilter>=7
-    libavformat>=58
-    libavcodec>=58
-    libswresample>=3
-    libswscale>=3
-    libavutil>=56
-    )
+  set(FFMPEG_FIND_COMPONENTS avdevice avfilter avformat avcodec avutil)
```

Comment:
is there a reason why swresample and swscale are removed?

Reply:
`swresample` and `swscale` implements algorithms on lawer-level.
`avfilter` library wraps them and provides high-level functions that we use.

`swresample` and `swscale` are required at runtime, but not during the build time as we do not include their header directly.